geonode/geoserver/helpers.py

```diff
@@ -638,13 +640,29 @@ def set_attributes(layer, overwrite=False):
             attribute_map = []
 
     elif layer.storeType in ["dataStore", "remoteStore", "wmsStore"]:
-        dft_url = re.sub("\/wms\/?$",
+        dft_url = ''
+        if 'geonode.geoserver' in settings.INSTALLED_APPS:
```

Comment:
Me too, this worries me ! ;-)
I wanted to talk about that during the standup today.
Geonode is of course full compatible with Geoserver as Geonode has been built over Geoserver. My question, if the user choose QGIS Server, should it be full compatible too, totally transparent for the admin/user whatever the backend ? For example, for now the QGIS project template works only with TIFF and shapefiles. But I can see in the code that there are various `store` in geonode : dataStore, remoteStore, wmsStore, gxp_arcrestsource (ESRI ?), PostGIS ... http://docs.geonode.org/en/master/tutorials/devel/geonode_apis/importer/formats.html